scipy/stats/tests/test_stats.py

```diff
@@ -6188,160 +6188,113 @@ def test_describe_empty(self, xp):
-                                   xp.asarray(0.04761502382843208),
-                                   xp.asarray(0.98880018772590561))
-    pv_skew_less, pv_kurt_less = 1 - pv_skew / 2, pv_kurt / 2
-    pv_skew_greater, pv_kurt_greater = pv_skew / 2, 1 - pv_kurt / 2
-    x = np.array((-2, -1, 0, 1, 2, 3.)*4)**2
-    x_xp = xp.asarray((-2, -1, 0, 1, 2, 3.)*4)**2
-    attributes = ('statistic', 'pvalue')
+class NormalityTests:
+    def test_too_small(self, xp):
+        # 1D sample has too few observations -> error
+        test_fun = getattr(stats, self.test_name)
+        message = "...requires at least..."
+        with pytest.raises(ValueError, match=message):
+            test_fun(xp.asarray(4.))
 
-    res = stats.normaltest(x_xp)
-    xp_assert_close(res.statistic, st_normal)
-    xp_assert_close(res.pvalue, pv_normal)
-    check_named_results(stats.normaltest(x), attributes)
-
-    res = stats.skewtest(x_xp)
-    xp_assert_close(res.statistic, st_skew)
-    xp_assert_close(res.pvalue, pv_skew)
-    res = stats.skewtest(x_xp, alternative='less')
-    xp_assert_close(res.statistic, st_skew)
-    xp_assert_close(res.pvalue, pv_skew_less)
-    res = stats.skewtest(x_xp, alternative='greater')
-    xp_assert_close(res.statistic, st_skew)
-    xp_assert_close(res.pvalue, pv_skew_greater)
-    check_named_results(stats.skewtest(x), attributes)
-
-    res = stats.kurtosistest(x_xp)
-    xp_assert_close(res.statistic, st_kurt)
-    xp_assert_close(res.pvalue, pv_kurt)
-    res = stats.kurtosistest(x_xp, alternative='less')
-    xp_assert_close(res.statistic, st_kurt)
-    xp_assert_close(res.pvalue, pv_kurt_less)
-    res = stats.kurtosistest(x_xp, alternative='greater')
-    xp_assert_close(res.statistic, st_kurt)
-    xp_assert_close(res.pvalue, pv_kurt_greater)
-    check_named_results(stats.kurtosistest(x), attributes)
-
-    # some more intuitive tests for kurtosistest and skewtest.
-    # see gh-13549.
-    # skew parameter is 1 > 0
-    a1 = stats.skewnorm.rvs(a=1, size=10000, random_state=123)
-    a1_xp = xp.asarray(a1)
-    pval = stats.skewtest(a1_xp, alternative='greater').pvalue
-    xp_assert_close(pval, xp.asarray(0.0, dtype=a1_xp.dtype), atol=9e-6)
-
-    # excess kurtosis of laplace is 3 > 0
-    a2 = stats.laplace.rvs(size=10000, random_state=123)
-    a2_xp = xp.asarray(a2)
-    pval = stats.kurtosistest(a2_xp, alternative='greater').pvalue
-    xp_assert_close(pval, xp.asarray(0.0, dtype=a2_xp.dtype), atol=1e-15)
-
-    # Test axis=None (equal to axis=0 for 1-D input)
-    res = stats.normaltest(x_xp, axis=None)
-    xp_assert_close(res.statistic, st_normal)
-    xp_assert_close(res.pvalue, pv_normal)
-
-    res = stats.skewtest(x_xp, axis=None)
-    xp_assert_close(res.statistic, st_skew)
-    xp_assert_close(res.pvalue, pv_skew)
-
-    res = stats.kurtosistest(x_xp, axis=None)
-    xp_assert_close(res.statistic, st_kurt)
-    xp_assert_close(res.pvalue, pv_kurt)
```

Comment:
Assume this is tested elsewhere as I couldn't see these tests in the updated version?